mergify_engine/actions/merge_base.py

```diff
@@ -99,7 +107,9 @@ def get_strict_status(
     ) -> check_api.Result:
 
         summary = ""
-        if self.config["strict"] in ("smart+fasttrack", "smart+ordered"):
+        if self.config["strict"] in [
```

Comment:
This now loop across all possible values instead of just "smart+fasttrack" and "smart+ordered".

Reply:
Is it better ?
```
if (
            self.config["strict"] == StrictMergeParameter["FASTTRACK"].value
            or StrictMergeParameter["ORDERED"].value
        ):
```

Reply:
1. That's not a valid syntax
2. That being said, I understand what you mean, and yes that's what the logic you want indeed
3. You should not use the `.value` attribute but e.g. `StrictMergeParameter.ORDERED` and that's it